clarity_ext/dilution.py

```diff
@@ -79,27 +78,38 @@ def _build_plate_sorting_map(containers):
         return plate_position_numbers
 
     def __str__(self):
-        return "<{type} {robot} {rows}x{cols}>".format(type=self.__class__.__name__,
-                                                       robot=self.robot_name,
-                                                       rows=self.plate_size_y,
-                                                       cols=self.plate_size_x)
-
-PLATE_TYPE_96_WELL = 1
```

Comment:
I prefer the rows and cols notation before height and width. This is also a notation that the lab-people use for plates and tube racks. I think however that the size_y and size_x should be removed.